python/FactorySystem/InputParameters.py

```diff
@@ -45,6 +45,11 @@ def isValid(self, name):
     def __contains__(self, item):
         return item in self.desc
 
+    def get(self, key, default=None):
```

Comment:
While this is useful. I've been trying to keep the interfaces between MOOSE and this class the same just os users can move between them and have the same interface. There isn't a method like this on the MOOSE side.

Reply:
Input parameters object implements the python dictionary operator (``[bla]`` indexing).  This method just makes that interface consistent with the python dictionary interface which also has a ``.get`` with the same semantics.  But I don't feel that strongly about it either way.

Reply:
Let's remove it. It's just a convenience and isn't necessary. The rest of this looks fine. I'll review and merge after that.
Thanks

Reply:
done.